src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/DashboardTopBanner/DashboardTopBannerContent.tsx

```diff
@@ -263,7 +262,7 @@ export const DashboardTopBannerContent = (props: DashboardTopBannerProps) => {
               )}
             </div>
             <a
```

Comment:
Q: SHould this be a `<Link>` or `<PageLink>` tag instead of an `<a>` since its an internal link?

```sh
git grep -Pn -A2 " <a\s?" src | grep -E " href=[\"'].*?[\"']" | grep -Ev "https?:"

src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/DashboardTopBanner/DashboardTopBannerContent.tsx-265-              href="/how-it-works"
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/View.tsx-365-                  href="/user/welcome/free-scan?referrer=dashboard"
src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/welcome-to-plus/WelcomeToPlusView.tsx-89-                          href="/how-it-works"
src/app/(proper_react)/(redesign)/(public)/breach-details/[breachName]/BreachDetailView.tsx:172:              <a href="#delayed-reporting">
src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Header.tsx-20-                  href="#data-brokers-section"
src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Header.tsx-26-                  href="#data-breach-section"
src/emails/email2022.js:162:        <a href='${links(data).legal}'>
src/emails/email2022.js:166:        <a href='${links(data).termsAndPrivacy}'>
src/emails/emailBreachAlert.js-42-          href='${ctaHref}'
src/emails/emailMonthlyUnresolved.js-116-          href='${data.ctaHref}'
src/emails/emailSignupReport.js-103-          href='${data.ctaHref}'
src/emails/emailVerify.js-37-          href='${data.ctaHref}'
```

Reply:
Updated it:
https://github.com/mozilla/blurts-server/pull/4632/commits/df119428bdb0f7d9f54414d52ea42bed66db8b58